tests/unit/ForeignJurisdiction.spec.ts

```diff
@@ -29,80 +29,124 @@ describe('ForeignJurisdiction', () => {
       })
     const vm: any = wrapper.vm
 
-    wrapper.find('#country-selector').trigger('click')
-    await Vue.nextTick()
-    wrapper.find('.menuable__content__active').findAll('.v-list-item').at(123).trigger('click')
+    await wrapper.find('#country-selector').trigger('click')
+    wrapper.findAll('.v-list-item').at(123).trigger('click')
 
     expect(vm.selectedCountry).toEqual({code: 'LB', name: 'Lebanon'})
     expect(vm.selectedRegion).toEqual({})
     wrapper.destroy()
   })
 
-  xit('set correct region when selected via the region selector', async () => {
+  it('set correct Canadian province when selected via the region selector', async () => {
     const wrapper = mount(ForeignJurisdiction,
       {
         vuetify
       })
     const vm: any = wrapper.vm
 
-    wrapper.find('#country-selector').trigger('click')
-    await Vue.nextTick()
-    wrapper.findAll('.v-list-item').at(39).trigger('click')
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(39).trigger('click')
+
+    const region = wrapper.find('#region-selector')
+    await region.setValue(JSON.stringify({code: 'ON', name: 'Ontario'}))
+
     expect(vm.selectedCountry).toEqual({code: 'CA', name: 'Canada'})
+    expect(vm.selectedRegion).toEqual(JSON.stringify({code: 'ON', name: 'Ontario'}))
+    wrapper.destroy()
+  })
+
+  it('no BC as a province when Canada is selected as the country', async () => {
+    const wrapper = mount(ForeignJurisdiction,
+      {
+        vuetify
+      })
 
-    await Vue.nextTick()
-    wrapper.find('#region-selector').trigger('click')
-    wrapper.find('.menuable__content__active').findAll('.v-list-item').at(7).trigger('click')
-    await Vue.nextTick()
-    expect(vm.selectedRegion).toEqual({name: 'Ontario', short: 'ON'})
+    await wrapper.find('#country-selector').trigger('click')
+    await wrapper.findAll('.v-list-item').at(39).trigger('click')
+
+    const provinces = wrapper.findAll('.v-select').at(1).props('items')
+
+    expect(!provinces.includes({'name': 'British Columbia', 'short': 'BC'}))
```

Comment:
Can this be written as `expect(provinces).includes(...` ?

Reply:
Hmm, this doesn't seem to work.

Reply:
Try `toContain()`.

Reply:
There's a code smell regarding this. I've tried  `toContain()` but it doesn't also seem to work. I'll be looking into this to fix it 👍

Reply:
There has to be some way to check if an array has a particular element...

https://jestjs.io/docs/expect